app/controllers/tasks_controller.rb

```diff
@@ -10,14 +10,25 @@ def update
       Action.update(action.id, completed: action_completed)
     end
 
+    @task.update(not_applicable: not_applicable)
+    unset_all_actions if @task.not_applicable?
+
     redirect_to(project_path(@task.project))
   end
 
   def action_params
```

Comment:
Wonder if this should be `task_params`? I named it `action_params` because it was related to actions, but maybe task would be a better name now, because it's the task that is N/A 🤔

Reply:
I am not even sure we need the strong params as we are not doing mass assignment, but I take the point - `task_params` feels better, it then reads:

`task_params[:completed_aciton_ids]`

Which feels good.